x/emissions/keeper/keeper.go

```diff
@@ -1607,28 +1607,38 @@ func (k Keeper) GetIdsOfActiveTopics(ctx context.Context, pagination *types.Simp
 	if err != nil {
 		return nil, nil, err
 	}
+	rng := new(collections.Range[uint64]).StartExclusive(start)
```

Comment:
would we maybe want inclusive instead of exclusive?

Reply:
`nextTopicId` is set after the value is added to the array, so if we make "inclusive" then we'd return the same value twice across 2 consecutive calls